data/json/items/containers.json

```diff
@@ -2791,5 +2801,189 @@
         "max_contains_weight": "500 g"
       }
     ]
+  },
+  {
+    "id": "ammunition_can_30",
+    "type": "GENERIC",
+    "category": "container",
+    "name": "M19A1 ammo can",
```

Comment:
These specific names are not useful to the average player, can they use the variant system to give them the toggled generic name?

(I always use the specific names tho <3)

*Applies also to M2A2, PA108 SAW, PA-120 ammo cans + M289 container

Reply:
Well i dont really know what generic name they can have, their non official names is in id - 30 cal, 50 cal, 50 fat and 40mm. If you have some suggestion I'll be glad to use it, we can put the official names in the description (or just remove it at all, usually people dont know their official names)

Reply:
I'd go for nonofficial name("7.62x51mm ammo can") w/ official in description.

Reply:
Thats a bad idea, they are reused in many ways, same irl and in the game. `ammunition_can_30` is used to store 9mm, 300blk and 300, when `ammunition_can_50` contain 223, 308 and 50.